src/utah-design-system/Button.stories.jsx

```diff
@@ -0,0 +1,94 @@
+import Button from './Button';
+
+export default {
+  title: 'Utah Design System/Button',
+  component: Button,
+};
+
+export const Default = () => (
+  <>
+    <h3>Outlined</h3>
+    <p>
```

Comment:
**thought**: adding a small gap would make this look a bit cleaner.

Reply:
57ba0cd69eb59df5987fb4a85bee36b9375a3754